modules/vaos/app/controllers/vaos/available_appointments_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_dependency 'vaos/application_controller'
+
+module VAOS
+  class AvailableAppointmentsController < ApplicationController
+    def index
+      response = systems_service.get_facility_available_appointments(
+        current_user, facility_id, start_date, end_date, clinic_ids
+      )
+
+      render json: VAOS::AvailabilitySerializer.new(response)
+    end
+
+    private
+
+    def systems_service
+      VAOS::SystemsService.new
+    end
+
+    def appt_params
+      params.require(:facility_id)
+      params.require(:start_date)
+      params.require(:end_date)
+      params.require(:clinic_ids)
+      params.permit(
+        :facility_id,
+        :start_date,
+        :end_date,
+        clinic_ids: []
```

Comment:
Would we be passing this in on the FE in as `clinic_ids[]`? Also, not sure if it matters, but the VAOS service takes this is a comma separated list of ids (because they needed to add yet another way of passing an array in a query param, I guess).

Reply:
From our FE it will the Rails way `../?clinic_ids[]=1&clinic_ids[]=2&clinic_ids[]=3`, I'll double check how we're sending it down to VAOS.